### ISSUE TITLE
Script error for "clustergrammer_widget"

### ISSUE THREAD

**Issue by IanMcDowell** [MaayanLab/clustergrammer-widget#4]
After installing and enabling the clustergrammer_widget as follows:

`jupyter nbextension enable --py --sys-prefix widgetsnbextension`
`Enabling notebook extension jupyter-js-widgets/extension...`
     ` - Validating: OK`

`jupyter nbextension enable --py --sys-prefix clustergrammer_widget`
`Enabling notebook extension clustergrammer_widget/extension...`
      `- Validating: OK`

The widget fails to work:

![screen shot 2017-12-21 at 9 40 01 am](https://user-images.githubusercontent.com/5889268/34260182-faf186f8-e632-11e7-9dba-865ecd58bbce.png)

Have you seen anything like this before?

Thanks

**Comment by cornhundred** [comment]
We have seen this error before and been able to fix it by uninstalling and reinstalling the Clustergrammer-Widget with pip (the latest Clustergrammer-Widget version was just released, v1.13.2). You might also want to try restarting the Jupyter notebook if you have not already. I'm also using notebook version 5.2.2, but I'm not sure if that makes a difference. What operating system are you using?

Let us know if that works. If not we can try and raise an issue in [ipywidgets](https://github.com/jupyter-widgets/ipywidgets)

**Comment by IanMcDowell** [comment]
I've fixed the error. While my discoveries may be of only limited use to other users, I'll share here anyway.

My notebooks are spawned through a JupyterHub (outside of my path) and while I have the hub configured to enable kernels to launch from my `--sys-prefix` location, nbextensions that run in those kernels are run from my `--user` location.

To fix my error, I had to run:
`jupyter nbextension install --py --user clustergrammer_widget`
`jupyter nbextension install --py --user widgetsnbextension`
`jupyter nbextension enable widgetsnbextension --user --py`
`jupyter nbextension enable widgetsnbextension --user --py`

Thanks for your help.
